docs/about/index.rst

```diff
@@ -1,13 +1,37 @@
 .. _about:
 
-About
-=====
+About Fairlearn
+===============
 
 .. _mission:
 
 Mission
 -------
 
+Fairlearn is an open source project that seeks to empower data scientists and
```

Comment:
What is the type of language we would like to use? In my opinion, expressions like "seeks to empower", "aspires to include", "firmly grounded in the understanding" sound a bit more formal than necessary, which I personally consider a bit less welcoming. But I can also imagine others might feel that's appropriate in a mission statement, so i'm happy to be convinced otherwise :)

Reply:
I completely agree for the first two (but I do like the third one). I'm pretty sure I didn't write that either :D This is a remnant from the pre-governance change days.